ArjunaJTS/narayana-jts-ibmorb/pom.xml

```diff
@@ -23,7 +23,22 @@
   <name>Narayana: ArjunaJTS narayana-jts-ibmorb</name>
   <packaging>jar</packaging>
   <description>JTS uber jar with ibmorb stubs</description>
+  <properties>
+    <propertiesFileName>jbossts-ibmorb-properties.xml</propertiesFileName>
+  </properties>
   <build>
+    <resources>
+      <resource>
+        <directory>${project.basedir}/src/main/resources</directory>
+        <filtering>true</filtering>
+      </resource>
+    </resources>
+    <testResources>
+      <testResource>
+        <directory>${project.basedir}/src/test/resources</directory>
+        <filtering>true</filtering>
+      </testResource>
+    </testResources>
```

Comment:
This should not be here. We do not need testResources AFAIU

Reply:
Here I was thinking if someone uses this dependency and they want to write a test, so in order to execute that test they would require `testResource` i.e. the build info file. So at that time user will have to explicitly include this resource in test target directory. To avoid that I added it here.

Reply:
I am not sure I follow you. I thought that [Maven Resources Plugin filters](https://maven.apache.org/plugins/maven-resources-plugin/examples/filter.html) items that has been indicated as resources (e.g. `<directory>${project.basedir}/src/main/resources</directory>`). Why tests should require a `testResource`?

Reply:
In the process of maven builds when different classes and test-classes are packaged inside `/target` folder, there are sub-folders created. One of them is `/target/classes` and another is `/target/test-classes`. So under `/target/classes` we have all the classes with their resources all packed together and in the same way in `/target/test-classes` contains all the test classes with their resources.

Now here comes `maven resource plugin filter` into picture. The main function of filtering is to replace maven related variables (like `${variable}`) with their value. The `<resources/>` will replace all the variables inside `/target/classes` folder resource files and `<testResources/>` will replace all the variables inside `/target/test-classes` folder resource files with their respective values.

So, until these resource file variables are not replaced with their values, the tests will fail.